task-2/src/components/Loader/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { Oval } from 'react-loader-spinner';
+
+import './index.css';
+
+export const LoaderSpinner = () => {
+	return (
+		<div className='loaderContainer'>
+			<h3>Loading All Movies ...</h3>
```

Comment:
This makes the LoaderSpinner component less reusable.
Consider moving the text explanation outside of the loader itself, because typically design systems reuse the same looking loaders everywhere, both page-size and inline inside buttons.

Reply:
Resolved!

Reply:
fixed.